### ISSUE TITLE
Simple Keras Model with k-fold cross validation

### ISSUE THREAD

**Issue by nkumar115** [nkumar115/Data-Science#16]
**TL;DR** 
Simple Keras Model with k-fold cross validation
k-fold Cross-Validation in Keras Convolutional Neural Networks

Links:

1. https://www.machinecurve.com/index.php/2020/02/18/how-to-use-k-fold-cross-validation-with-keras/#
2. https://www.kaggle.com/stefanie04736/simple-keras-model-with-k-fold-cross-validation

3. https://medium.com/@navmcgill/k-fold-cross-validation-in-keras-convolutional-neural-networks-835bed559d04

Takeways:
Above three tutorials are good to understand K-Fold cross validation for keras deep learning.
Last one has used **pretrained** **BERT** **model** for word to vec transformation 

**ScreenShot**
![image](https://user-images.githubusercontent.com/29604715/80741433-3f51a980-8b37-11ea-864a-864ee494c6c7.png)
